bin/wazo-service

```diff
@@ -108,17 +108,17 @@ start_monit() {
 
 close_sip_port() {
     echo "Closing port 5060."
-    iptables -n --list | grep -q '^DROP.*5060'
+    nft -n list table filter | grep -q "^\s*udp dport 5060 counter packets 0 bytes 0 drop"
```

Comment:
Hum I don't think you can match on `0 bytes 0 drop`. These number will change if packets are dropped
For fun, here's the jq version
```bash
nft -j list table filter | jq -e -r '.nftables[]
  | select(.rule.chain == "INPUT")
  | .expr[].match
  | select((.left.payload.field == "dport") and (.right == 5060))'
```

Reply:
You're quite right. We could use `grep "^\s*udp dport 5060 counter .* drop"` or something like it though.